insights/core/evaluators.py

```diff
@@ -288,11 +288,11 @@ def run_reducers(self, metadata=None):
             self.parser_results,
             self.all_output,
             self.handle_reducer_error(),
-            stats=self.stats['reducer'])
+            reducer_stats=self.stats['reducer'])
```

Comment:
OK, will remove this change and will modify the `master` after this PR is merged.